src/app/workbench/WorkbenchGenericView.tsx

```diff
@@ -208,6 +208,7 @@ export function WorkbenchGenericView({
                   {/* TODO refactor view header such that the logic (using hooks) and the visual representation are separated */}
                   <ViewChooser
                     views={chooserOptions}
+                    selectedView={viewPlugin?.desc}
```

Comment:
I was reviewing this and saw some hard-coded idTypes that should be refactored in a follow-up issue https://github.com/datavisyn/reprovisyn/issues/398.